package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "leaflet-distortableimage",
-  "version": "0.17.7",
```

Comment:
@jywarren i think this was a big issue - we were updating to an unreleased version.